src/utils/image_plugins/chart.ts

```
@@ -26,5 +26,29 @@
   return imagePath;
 };
 
+const dumpHtml = async (params: ImageProcessorParams) => {
+  const { beat } = params;
+  if (!beat.image || beat.image.type !== imageType) return;
+
+  const chartData = JSON.stringify(beat.image.chartData, null, 2);
+  const title = beat.image.title || "Chart";
+  const chartId = `chart-${Math.random().toString(36).substr(2, 9)}`;
```

ESLint

[error] Make sure that using this pseudorandom number generator is safe here.

(sonarjs/pseudo-random)